ErpNet.FP.Core/Drivers/BgIslFiscalPrinter.Frame.cs

```diff
@@ -148,6 +149,8 @@ protected virtual byte[] BuildHostFrame(byte command, byte[]? data)
                     if (wait)
                     {
                         // The FiscalPrinter is still not ready, so make another read
+                        if (command == CommandFiscalReceiptTotal && data != null && data.Length > 0 && data[0] == '2')
```

Comment:
This is kind of weird. According to the manual (PM_CommunactionPrococol_v2.10_eng.pdf, command 35h returns

byte 0 - error code, negative
byte 1 - status, 'D' or 'R'
byte 2 - sum tendered

Reply:
This is about SEND data. Sending  "CommandFiscalReceiptTotal"  with data "2" means "Pay by Card and use payment terminal". Only then we must remove limit of Read retries (because this can break transaction).